packages/asset-swapper/test/swap_quote_calculator_test.ts

```diff
@@ -78,6 +78,7 @@ describe('swapQuoteCalculator', () => {
                         new BigNumber(200),
                         0,
                         false,
+                        false,
```

Comment:
Tests for the true condition would be nice.

Reply:
Yeah, will be adding coverage in a seperate PR